tests/hikari/impl/test_entity_factory.py

```diff
@@ -2034,6 +2036,7 @@ def test_deserialize_guild_stage_channel_with_null_fields(self, entity_factory_i
                 "user_limit": 3,
                 "rtc_region": None,
                 "type": 6,
+                "last_message_id": None,
```

Comment:
Should prob add `assert channel.last_message_id is None` to this test case and `test_deserialize_guild_stage_channel_with_unset_fields`

Reply:
done :)